hub/powertoys/grouppolicy.md

```diff
@@ -93,6 +93,7 @@ These policies have a higher priority than, and will override, the policy "Confi
 
 | Utility | ADMX GP name |ADMX GP unique name /</br>Registry value name /</br>Intune PolicyID |
 | :--- | :--- | :--- |
+|Advanced Paste|Advanced Paste: Configure enabled state|ConfigureEnabledUtilityAdvancedPaste|
```

Comment:
Please remove the old one from "Paste as Plaintext". Or should we add a `supported from ... to ... note`?

Reply:
Good spotting! Just pushed. Thank you!